credentials/alts/internal/handshaker/handshaker.go

```diff
@@ -74,8 +74,10 @@ func init() {
 	}
 }
 
-func acquire(n int64) bool {
+func acquire() bool {
```

Comment:
Does this work for you: https://godoc.org/golang.org/x/sync/semaphore?

Reply:
That works. Code is changed.